### PR TITLE
Milestone 2: Components

### PR DESCRIPTION
**Pull request by idgm5**
I created the basic components for the project which includes the classes to display the calculator buttons and the final result as the requirements suggested.

**Requirements**
1. Prepare the directory structure
2. Implement the App component
3. Implement the Display component
4. Implement the Button component
5. Implement the ButtonPanel component

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Button extends React.Component {
+  render() {
+    return <button>{this.props.buttonName}</button>;
+  }
+}
+
```
Comment (shloch) [inline src/components/Button.js]: `buttonName` should be string, enforce this with `propTypes` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png)]

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Display extends React.Component {
+  render() {
+    return <h1>{this.props.result}</h1>;
+  }
+}
+
```
Comment (shloch) [inline src/components/Display.js]: `result` should be string (enforce with `propTypes`) [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)]

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @idgm5 , 

Great job on what you've done
Please let's include small additions mentioned below (prior to getting a thumbs up) 

### Happy coding

**Review by bellom — APPROVED** [review]
## Hi @idgm5  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request closed without merging**